features/api_services_list.feature

```diff
@@ -0,0 +1,395 @@
+              "name": "anbox-cloud"
+            },
+            {
+              "available": false,
+              "description": "Common Criteria EAL2 Provisioning Packages",
+              "entitled": true,
+              "name": "cc-eal"
+            },
+            {
+              "available": true,
+              "description": "Expanded Security Maintenance for Applications",
+              "entitled": true,
+              "name": "esm-apps"
+            },
+            {
+              "available": true,
+              "description": "Expanded Security Maintenance for Infrastructure",
+              "entitled": true,
+              "name": "esm-infra"
+            },
+            {
+              "available": false,
+              "description": "NIST-certified FIPS crypto packages",
+              "entitled": true,
+              "name": "fips"
+            },
+            {
+              "available": false,
+              "description": "Preview of FIPS crypto packages undergoing certification with NIST",
+              "entitled": true,
+              "name": "fips-preview"
+            },
+            {
+              "available": false,
+              "description": "FIPS compliant crypto packages with stable security updates",
+              "entitled": true,
+              "name": "fips-updates"
+            },
+            {
+              "available": true,
+              "description": "Management and administration tool for Ubuntu",
+              "entitled": true,
+              "name": "landscape"
+            },
+            {
+              "available": true,
+              "description": "Canonical Livepatch service",
+              "entitled": true,
+              "name": "livepatch"
+            },
+            {
+              "available": true,
+              "description": "Ubuntu kernel with PREEMPT_RT patches integrated",
+              "entitled": true,
+              "name": "realtime-kernel"
+            },
+            {
+              "available": false,
+              "description": "Security Updates for the Robot Operating System",
+              "entitled": true,
+              "name": "ros"
+            },
+            {
+              "available": false,
+              "description": "All Updates for the Robot Operating System",
+              "entitled": true,
+              "name": "ros-updates"
+            },
+            {
+              "available": false,
+              "description": "Security compliance and audit tools",
+              "entitled": true,
+              "name": "usg"
+            }
+          ]
+        },
+        "meta": {
+          "environment_vars": []
+        },
+        "type": "ServicesList"
+      }
+      """
+
+    Examples: ubuntu release
+      | release | machine_type  |
+      | xenial  | lxd-container |
+      | bionic  | lxd-container |
+      | focal   | lxd-container |
+      | jammy   | lxd-container |
+      | noble   | lxd-container |
```

Comment:
Please add oracular to the list so we can test this works on interim releases too